api-authorization/server/server.js

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable no-unused-vars -- Remove me! */
+import 'dotenv/config';
+import pg from 'pg';
+import argon2 from 'argon2';
+import express from 'express';
+import jwt from 'jsonwebtoken';
+import { ClientError, errorMiddleware, authMiddleware } from './lib/index.js';
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false,
+  },
+});
+
+const app = express();
+app.use(express.json());
+
+app.post('/api/auth/sign-up', authMiddleware, async (req, res, next) => {
```

Comment:
The `auth` endpoints should _not_ require authorization since the user has not yet signed up or signed in, so they do not have any authentication tokens.